### ISSUE TITLE
Sign-in page not mobile friendly

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#12875]
### System details

    RStudio Edition : Server
    RStudio Version : v2023.03.0+386
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem

- try to login to RStudio Server from a mobile device

### Describe the problem in detail

- the page is shown with tiny text by default, and you have to zoom to use it
- note that Safari on iOS has an auto-zoom feature that may zoom in on the input fields by default, but this is non-standard behaviour and shouldn't be relied on (e.g. doesn't happen on Chrome on Android):

<img width="355" alt="Screenshot of sign-in page on an Android phone" src="https://user-images.githubusercontent.com/10569626/226713436-f884fbf1-009f-4bd4-bb6d-81ce01f295fa.png">

- on RStudio Workbench/Pro, the sign-in page has additional text in the page banner that isn't responsive, so if you zoom in to a tablet/phone size, the header gets garbled:

<img width="692" alt="Screenshot of zoomed-in sign-in page with messed up formatting" src="https://user-images.githubusercontent.com/10569626/226723360-dcadb925-42aa-4588-8c5a-d7eda419b6f5.png">

### Describe the behavior you expected

- sign-in page should have a meta viewport set to make this mobile friendly
- header should display correctly when zoomed in even with RStudio Pro

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
Decided this makes more sense in the pro repo, opened https://github.com/rstudio/rstudio-pro/issues/4472.
